### PR TITLE
Fix the `hideElements` option

### PR DESCRIPTION
**Pull request by krnik**
Should close: https://github.com/sindresorhus/capture-website/issues/43

### CODE REVIEW

**Comment by sindresorhus** [comment]
Ensure running `$ npm test` locally passes.

**Comment by krnik** [comment]
Sorry, I am sure I did run the test command but maybe I did that before I saved any changes.

File: index.js [hunk 1]
```diff
@@ -11,12 +11,6 @@ const writeFile = promisify(fs.writeFile);
...
 
-const hideElements = elements => {
-	for (const element of elements) {
-		element.style.visibility = 'hidden';
-	}
-};
-
 const removeElements = elements => {
 	for (const element of elements) {
 		element.style.display = 'none';
```
Comment (Kikobeats) [inline index.js:16]: @krnik @sindresorhus 

should `removeElements` follows the same approach using `page.addStyleTag`?

Comment (krnik) [reply index.js:16]: Yes, I think so but just so the logic is consistent between those two options ;)

Comment (sindresorhus) [reply index.js:16]: Yes. I also just realized that we probably should also add `!important` to both of these, so we're sure nothing can override them.

**Pull request merged**